src/routes/project/TaskRow.svelte

```diff
@@ -144,4 +175,19 @@
 		border-radius: 50%;
 		overflow: hidden;
 	}
+	.sticky-name {
+		position: absolute;
+		top: auto;
+		left: 0;
+		padding: var(--padding-bars) 2rem 0;
```

Comment:
The padding here should be the same as in `.task__detail-name`, where it is done with `top: var(--padding-bars);`. Instead of relying on that 2rem padding, we should change offset parent, which currently is `body` but it should be a container for this thing, which is the `.background` div in `+page.svelte`. Which mean that one should have `position: relative` and margins instead of paddings. (and, I think, its time to rename it into, something like `.task-container`, because it doesn't play background role anymore (and, its background could be safely removed).